raiden-ts/src/epics.ts

```diff
@@ -167,8 +168,9 @@ export function raidenRootEpic(
     // wire deps.latest$ when observableFactory below gets subscribed, and tears down on complete
     () => getLatest$(limitedAction$, limitedState$, deps).subscribe(deps.latest$),
     () =>
-      // like combineEpics, but completes action$, state$ & output$ when a raidenShutdown goes through
-      from(Object.values(RaidenEpics)).pipe(
```

Comment:
Will the epics be restarted? Or will the user need to reload?

Reply:
This is a fatal, unexpected, non-recoverable error. It'll shutdown for good, and the user needs to reload when this happens.